docs/glossary.rst

```diff
@@ -27,6 +27,14 @@
 
       https://pypi.python.org/pypi/asyncio/
 
+   brotlipy
```

Comment:
FYI we switched from `brotlipy` to `brotli` a year ago https://github.com/aio-libs/aiohttp/pull/3803

Do you have a reason for the reverse direction?

Reply:
brotlipy was mentioned in #5219 
But actually it checked against Brotli package.
https://github.com/derlih/aiohttp/blob/a58b7ed4a4d85a9e5d530e3c99c685455bae65e4/aiohttp/http_parser.py#L29-L34